codex-rs/tui/src/cli.rs

```diff
@@ -17,6 +17,12 @@ pub struct Cli {
     #[arg(long, short = 'm')]
     pub model: Option<String>,
 
+    /// Convenience flag to select the local open source model provider.
+    /// Equivalent to -c model_provider=oss; verifies a local Ollama server is
```

Comment:
should really be a macro for `--profile oss` so you can have a default model for your `oss` profile that differs from the default model for your default profile